worldedit-bukkit/src/test/java/com/sk89q/wepif/TestOfflinePermissible.java

```diff
@@ -185,4 +188,94 @@ public Location getBedSpawnLocation() {
     public Map<String, Object> serialize() {
         throw new UnsupportedOperationException("Not supported yet.");
     }
+
```

Comment:
Curious if this might be easier to do with a mock or a Proxy.

Reply:
maybe in the long run.